test/fixtures/basic.bad.css

```diff
@@ -19,6 +19,12 @@
     color: red;
 }
 
+@media (width: 100px) and (width: 60rem) {
```

Comment:
Could you change to min-width and max-width respectively? Also, add another for height as well.